CHANGELOG.md

```diff
@@ -2,6 +2,21 @@
 
 All notable changes to this project will be documented in this file.
 
+## 2022-05-22
+
+* Updated ci-common-v2 to CentOS-7.9 and CUDA-11.4
+* Updated 2022 Packages
+  * Imath 3.1.5 (from 3.1.3)
+  * OpenColorIO 2.1.1 (from 2.1.0)
+  * OpenImageIO 2.3.14.0 (from 2.3.9.1)
```

Comment:
If you really want to be up to date, I think 2.3.15.0 is the latest (for 6 more days, then it will be .16.0, but that's not tagged yet).

Reply:
Just updated to 2.3.16.0!